Ihor branch

mykytapol opened the pull request:
Fixed the size of chapsumm and lookbox classes.
Set the margin-left and margin-right to 5%.
Set the width same as the text.

mykytapol commented:
Table width and border line fixed.
In order to let the table overflow to the right, we need to add a new class to that table.
I think this needs some change with Latex.

Ihor-Kon reviewed (commented):
To adjust the size of Figures 2.4 and 2.5 to take up the full width, need to add new class to it.

mykytapol commented:
The result for your last feedback.
1. Fixed.
2. Fixed.
3. Fixed.
4. Fixed.
5. Fixed.
6. Did not. We should assign new class to the tables which need some customization.
7. Fixed.
8. This also needs adding new style to the figures and then I will add style to that class.
